source/data-modeling/field-types.txt

```diff
@@ -0,0 +1,558 @@
+
+The following example creates a ``Voter`` class and specifies the
+``registered_at`` property as a ``Time`` field:
+
+.. literalinclude:: /includes/data-modeling/field-types.rb
+   :language: ruby
+   :start-after: # start-time
+   :end-before: # end-time
+
+.. note::
+
+   Storing a ``Date`` or ``DateTime`` value in a field specified as ``Time``
+   converts the value to ``Time`` when assigned. If you store a string in a
+   ``Time`` field, {+odm+} parses the string by using the ``Time.parse`` method.
+
+Date
+~~~~
+
+You can store the following value types in a field specified as a ``Date``:
+
+- ``Date``: Stores the value as provided.
+- ``Time``: Stores the date portion of the value in the value's time zone.
+- ``DateTime``: Stores the date portion of the value in the value's time zone.
+- ``ActiveSupport::TimeWithZone``: Stores the date portion of the value in the
+  value's time zone.
+- ``String``: Stores the date specified in the string.
+- ``Integer``: Takes the value as if it is a UTC timestamp and
+  converts it into your application's configured time zone. {+odm+} then stores
+  the date taken from that timestamp.
+- ``Float``: Takes the value as if it is a UTC timestamp and
+  converts it into your application's configured time zone. {+odm+} then stores
+  the date taken from that timestamp.
+
+Because converting a ``Time`` or ``DateTime`` discards the time portion, we
+recommend explicitly converting ``String``, ``Time``, and ``DateTime``, objects
+to ``Date`` before assigning them to the field.
+
+.. note::
+
+   When a database contains a string value for a ``Date`` field, the driver
+   parses the value by using the ``Time.parse`` method, then discards the time
+   portion. ``Time.parse`` considers values without time zones to be in local
+   time.
+
+DateTime
+~~~~~~~~
+
+When you assign a value to, or query on a ``DateTime`` field, {+odm+} converts
+the value to a UTC ``Time`` value before sending it to the MongoDB server.
+{+odm+} saves the value with the time zone embedded in the ``DateTime`` object. When
+you retrieve the value, {+odm+} converts the UTC time to the time zone
+configured for your application.
+
+The following example creates a ``Ticket`` class and specifies the ``opened_at``
+field as a ``DateTime`` field:
+
+.. literalinclude:: /includes/data-modeling/field-types.rb
+   :language: ruby
+   :start-after: # start-datetime
+   :end-before: # end-datetime
+
+If you save an integer or float value to a ``DateTime`` field, the value is treated as
+a Unix timestamp in UTC. The following example saves an integer value to the
+``opened_at`` field:
+
+.. literalinclude:: /includes/data-modeling/field-types.rb
+   :language: ruby
+   :start-after: # start-datetime-int
+   :end-before: # end-datetime-int
+
+If you save a string value to a ``DateTime`` field, {+odm+} saves the ticket
+with the time zone specified. If a time zone is not specified, {+odm+} saves the
+value by using the default {+odm+} timezone:
```

Comment:
S: what is the default timezone? does it use the one configured in your application?